### ISSUE TITLE
At cifar10 IDN noise rate setting, cann't repeat the results reported at paper.

### ISSUE THREAD

**Issue by zhengzhang37** [Justherozen/ProMix#6]
The cifar10 IDN 0.2 and 0.4 noise rate, I can only get about 93.6%-94% acc useing the code, and I can't find the reason. Could you tell me which hyper-parameters you use to train?

Thank you!

**Comment by Justherozen** [comment]
Thanks for your attention! The data loader of IDN noise follows previous works of learning with IDN, which will be different from that of symmetric/asymmetric noise. As for the parameters, we adopt rho=0.7 and 0.5 for the noise ratio of 0.2 and 0.4 respectively. For class-wise small-loss selection (CSS), 'bs_j' is set as 'len(indices)' and we adopt the original labels in the first 50 epochs and the predicted labels in the subsequent to separate each class-wise subset. Most of the parameters should be consistent with the original settings (we originally tended to prove the feasibility of ProMix for IDN noise and did not spend time tuning parameters).
